react-renderer/src/inspector/views/IViewProps.tsx

```diff
@@ -7,5 +7,6 @@ export interface IViewProps {
   frame: State | null;
   // Either a valid index in History
   frameIndex: number;
+  modShapes?(state: IState): void; // todo - null check
```

Comment:
why is it nullable?

Reply:
Not sure... I'll change it.

Reply:
Addressed in commit 2b9315a4a27f35fb751ac317f15cdbf9a18ec5e4